docs/docs/tutorial/part-1/index.mdx

```diff
@@ -0,0 +1,299 @@
+---
+title: Create and Deploy Your First Gatsby Site
+---
+
+import { Announcement } from 'gatsby-interface'
+
+## Introduction
+
+Now that you've set up your computer with all the tools you'll need, it's time to get started!
+
+In this part of the Tutorial, you will go through the process of creating your first Gatsby site and deploying it online for everyone to see.
+
+The diagram below shows a high-level view of how all the pieces of this process fit together. (Don't worry if this doesn't make sense yet. You'll learn about each step as you go.)
+
+![The workflow for how your code gets from your computer to your users. Extended description below.](deployment-workflow.png)
+
+<details>
+
+<summary>
+
+*Expand for detailed description* 👈
+
+</summary>
+
+First, you write the code for your Gatsby site from your computer. When you're ready to make your changes live on the internet, you go through the following steps:
+
+1. You push your changes from your computer to a remote repository on GitHub. GitHub is an online platform for storing code for your projects.
+1. Gatsby Cloud watches your GitHub repository for changes. When it sees your new changes, Gatsby Cloud builds your site from your code on GitHub.
+1. Gatsby Cloud hosts the finished version of your site at a unique URL, which users can use to access the latest version of your site.
+
+</details>
+
+
+## Create a Gatsby site
+
+To create your first Gatsby site, you're going to use a command from the Gatsby command line interface (CLI): `gatsby new`. This command brings up an interactive prompt that asks you questions about the site you want to build. After you enter all the information, the CLI uses your answers to automatically generate your new Gatsby site.
+
+![The welcome message for the interactive "gatsby new" command.](./gatsby-new-cli.png)
+
+Let's take a closer look at the process:
+
+1. Open the command line, and use the `cd` command to change directories into the folder where you want to create your new Gatsby site. For example, if you wanted to create your new site on your desktop, you might type:
+```shell:title=~
+cd Desktop
+```
+
+2. Run the following command from the command line. This will start up the interactive prompt to help you create a new Gatsby site.
+```shell:title=~/Desktop
+gatsby new
+```
```

Comment:
This is specific to Mac and not applicable directly in other environments. So not sure how valuable this is for people not on Mac. Can we maybe make this more generic so that it applies to everyone?

Reply:
Which piece? I thought that `cd` also worked on Windows?

Reply:
That there is a `~` and the desktop is called `Desktop`.

e.g. on Windows it would be `/c/Users/USERNAME/Desktop` or wherever they have their Desktop (it can be on another drive than C)